master/src/main/java/org/apache/celeborn/service/deploy/master/SlotsAllocator.java

```diff
@@ -135,13 +137,19 @@ static class UsableDiskInfo {
           workers, partitionIds, shouldReplicate, shouldRackAware, availableStorageTypes);
     }
 
+    int diskGroupCount = conf.masterSlotAssignLoadAwareDiskGroupNum();
+
     if (!initialized) {
-      initLoadAwareAlgorithm(diskGroupCount, diskGroupGradient);
+      initLoadAwareAlgorithm(diskGroupCount, conf.masterSlotAssignLoadAwareDiskGroupGradient());
     }
 
     Map<WorkerInfo, List<UsableDiskInfo>> restrictions =
         getRestriction(
-            placeDisksToGroups(usableDisks, diskGroupCount, flushTimeWeight, fetchTimeWeight),
+            placeDisksToGroups(
+                usableDisks,
+                diskGroupCount,
+                conf.masterSlotAssignLoadAwareFlushTimeWeight(),
+                conf.masterSlotAssignLoadAwareFetchTimeWeight()),
```

Comment:
Getting a config value from Celeborn Conf is expensive. Save the value and reuse would be better.